src/index.js

```diff
@@ -1,11 +1,25 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './css/index.css';
-import App from './App';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+import App from './components/App';
+import rootReducer from './reducers/index';
 
+const initialState = {
+  books: [
+    { id: Math.random(), title: 'Oliver Twist', category: 'Fiction' },
+    { id: Math.random(), title: 'Peter Pan', category: 'novel' },
+    { id: Math.random(), title: 'Superman', category: 'Sci-fi' },
```

Comment:
This is not a good way to initialise your state, the id changes on every refresh which name not be effificent.

Reply:
I had to do it this way as using `Math.random()` as value for `id` for this milestone is part of the project requirement.